README.md

```diff
@@ -438,12 +438,24 @@ The available commands are:
 - `l` or `list`: List all installed packages. To list installed packages, use the following command: `sympkg l`
 - `u` or `update`: Update an installed package. To update a package, use the following command: `sympkg u <package>`
 
+#### Usage example for local installation.
+The parent folder of the repo must be named < username >
+```bash
+$> sympkg i <username>/<repo_name> <path_to_repo>
+```
+
+```bash
+$> sympkg i somename/cool_repo /home/user/symai_projects/somename
```

Comment:
Did you test this? How would you distinguish between a GitHub URL and a local path? I did not see this check in the code nor do I see the extra param here. Please write tests that also verify both cases.

Reply:
If local path is not provided it will assume that it is a GitHub repo and will concatenate the GitHub url to the repo name